SpatialGDK/Extras/git/win/post-merge

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)"
+check_run() {
```

Comment:
I honestly think we can inline that. no need for an extra `check_run` function. 
thinking about it: the mac and windows post-merge scripts are very similar. (I think the only difference is whether we call `Setup.sh` or `Setup.bat`) we could check if we could make them one script instead of two

Reply:
Agree, just copied this directly across from the Setup.bat though to avoid making too many more changes on top of the 300 files, might as well though I guess 😂